app/pulp/app/settings.py

```diff
@@ -28,6 +29,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+# Silenced warnings
+# - fields.W342 is the warning for having "unique=True" on a ForeignKeyField, which is something
+#   we explicitly want, so we can enforce a single Importer per-repository in 3.0, but easily
+#   allow for multiple importers per-repository in a later version by lifting the contraint.
+SILENCED_SYSTEM_CHECKS = ["fields.W342"]
```

Comment:
This line could easily be forgotten when multiple importers is implemented. I think it makes sense to write that story if it doesn't already exist and mention this line on the ticket.